From: Pavel Sobolev <contact@paveloom.dev>
Subject: [PATCH] Skip failing tests.

The "album_art" tests require network access. The rest expect the
settings schema to be already installed. `RUST_TEST_THREADS=1` makes it
so that the name of the currently running test is actually printed.

Signed-off-by: Pavel Sobolev <contact@paveloom.dev>

--- a/src/meson.build
+++ b/src/meson.build
@@ -61,9 +61,22 @@ test(
     cargo_options,
     '--',
     '--nocapture',
+    '--skip',
+    'album_art::test',
+    '--skip',
+    'recognizer::provider::settings::tests',
+    '--skip',
+    'song_filter::tests',
+    '--skip',
+    'song_list::test',
+    '--skip',
+    'song_sorter::tests::compare',
+    '--skip',
+    'window::history_view::test',
   ],
   env: [
     'RUST_BACKTRACE=1',
+    'RUST_TEST_THREADS=1',
     cargo_env
   ],
   timeout: 300, # give cargo more time
